### PR TITLE
New models + examples of words in readme

### PR DESCRIPTION
**Pull request by annaproxy**
This commit is mainly for two new models and their accompanying datases.

- City name model: one for generating Dutch plaatsnamen (I scraped this data from Wikipedia for a **very** similar project a few months ago, so I am excited to reuse it here!) 
- Last name model: one for generating Dutch family names. [(Data from Meertens KNAW)](https://github.com/digitalheir/family-names-in-the-netherlands)

Both models can be loaded, for the last name model a new vocabulary object was made (without numbers, including some special alphabetic characters).

I moved the actual code for training a model to a file in the root directory called train_wrapper.py. 
Let me know if there is a better directory to put the training file in, without needing import hacks like `sys.path.append(..)`....

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,32 @@
...
 Ever wanted to generate new Dutch slang? Look no further! This application makes use of Recurrent Neural Networks trained on a comprehensive database of Dutch slang to generate new slang. Some words may immediately bring up a visceral feeling of what they mean, and this application allows you to provide the community with your interpretation of what the newly created slang means.
 
+## Examples
+We have several pre-trained models available to generate novel words from multiple classes.
+
+### Straattaal
+- rampa
+- joeko
+- opjo
+- oggor
```
Comment (Sasafrass) [inline README.md:12]:
```suggestion
- daggie
```
I really love daggie, and oggor is not my favorite

Comment (Sasafrass) [reply README.md:12]: Maybe we should also add our attributed meanings and "challenge people to come up with something better" ?

File: README.md [hunk 2]
```diff
@@ -2,6 +2,32 @@
...
+- zittie
+
+### Plaatsnamen
+- Boschem
+- De Heel
```
Comment (Sasafrass) [inline README.md:17]: This one is just so creative lol. I for one would definitely believe that this was a town in Limburg or het Rivierengebied

File: train_wrapper.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+from app.ml_models.rnn.vocabulary import Vocabulary
+from app.ml_models.rnn.train import train
+from app.ml_models.rnn.rnn_model import RNNAnna
+
+
+def run():
+    """Run a simple training loop."""
+    v = Vocabulary()
+    v.build(prefix=".", filename_source="data/familienamen.txt",filename_destination="vocabulary_extended.txt")
+    dataset = WordLevelDataset(prefix="data/", filename_datasets=["familienamen.txt"], vocabulary=v)
```
Comment (Sasafrass) [inline train_wrapper.py:16]: Should we maybe make this a little bit more generic and give the filename_source and similarly the filename_datasets as arguments to the run function or even as argparse (default) arguments if we only run this manually from the command line?

File: train_wrapper.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    # Currently only batch size 1 works
+    train_loader = DataLoader(dataset, 1, shuffle=True)
+    rnn = RNNAnna(dataset.vocabulary.size, hidden_size=128)
+    train(
+        rnn,
+        train_loader,
+        dataset,
+        epochs=5,
+        device="cuda",
+        name="familienamen",
```
Comment (Sasafrass) [inline train_wrapper.py:26]: Same as the above comment wrt making this argument more generic

File: README.md [hunk 5]
```diff
@@ -2,6 +2,32 @@
 
 Ever wanted to generate new Dutch slang? Look no further! This application makes use of Recurrent Neural Networks trained on a comprehensive database of Dutch slang to generate new slang. Some words may immediately bring up a visceral feeling of what they mean, and this application allows you to provide the community with your interpretation of what the newly created slang means.
 
+## Examples
+We have several pre-trained models available to generate novel words from multiple classes.
```
Comment (Sasafrass) [inline README.md:6]:
```suggestion
We have several pre-trained models available to generate novel words from multiple classes. The following is a selection of some of our favorite generated words per category. Can you come up with a better meaning for the slang words?
```

File: train_wrapper.py (hunk 3 above)
Comment (annaproxy) [reply train_wrapper.py:16]: Yeees  I did put a TODO: argparse there, right?

File: README.md (hunk 1 above)
Comment (annaproxy) [reply README.md:12]: Agree with daggie. I have no creativity to add definitions right now. should we format them like 

- **rampa**: een grote ramp

+ which language?

Comment (Sasafrass) [reply README.md:12]: Yes, I like the format. 
I'd say we should give both the Dutch and English translation of the meaning. May be able to work on it tomorrow morning, so I created this issue (https://github.com/Sasafrass/straattaal/issues/59). I pushed it to the new Documentation board, so it's no longer a blocker for this PR, and we can perhaps create a new PR for it.

File: train_wrapper.py (hunk 3 above)
Comment (Sasafrass) [reply train_wrapper.py:16]: Fair, I skimmed over that too quickly.

**Review by Sasafrass — APPROVED** [review]

**Pull request merged**